### ISSUE TITLE
Address Lack of Landmarks 

### ISSUE THREAD

**Issue by jenndiaz** [wwcodecolorado/scrollodex#96]
#### Description
The project does not meet best practice or WCAG AA when it comes to landmarks. This is an accessibility concern since screen reader users often use landmarks to navigate a page. 

This can be addressed by using semantic html such as `<main>` or with an aria role attribute`<role="main">` 
See resources below for more on ways to address this

#### How to visualize page structure with Wave 
1. install wave web accessibility evaluation tool browser extension 
2. run tool (on Mac `cmd + shift + u`) 
3. within summary sidebar, select structure tab 

#### Tasks
- [ ] All text content should be with in a landmark 
- [ ] There should be atleast one main landmark 

#### Resources 
[ARIA Landmarks Example](https://www.w3.org/WAI/ARIA/apg/example-index/landmarks/main.html)
[WAI-ARIA Docs on main role](w3.org/TR/wai-aria-1.2/#main)
[Deque Checklist on Landmarks](https://dequeuniversity.com/checklists/web/landmarks)

**Comment by heretoshleigh** [comment]
@jenndiaz Hey! I think I accidentally addressed this in PR #138. I meant for that PR to be limited to card styling, but I was playing around with some accessibility stuff on the same branch after I submitted the PR and didn't realize the subsequent commits would get pulled in (which there were several of because I also made and reverted a mistake). Ack, sorry for the confusion!

All I did was wrap the CardsDisplay component in a <main> tag in App.js. If I understand the issue correctly from my research, this is all we need, since we already have <header> and <footer> elements, which wrap all the other text and serve as banner and contentinfo landmarks. But if additional work is needed, happy to continue researching this. Thank you for the opportunity to learn about it!

**Comment by jenndiaz** [comment]
@heretoshleigh can confirm adding the <main> addresses this!! 

but I think we could we get rid of the extra div now.  This would clean up the code some and avoid using a non semantic tag. 

  ```
  <main>
        <div>
          <CardsDisplay />
        </div>
      </main>
```

I'll leave the issue open for now, if you take a look at it and can confirm it can be removed feel free to put up another PR removing it and link this issue.

**Comment by heretoshleigh** [comment]
@jenndiaz Thanks, Jenn! I took a look and see no problem with removing the extra div. Just submitted the PR.
